triggers/messageUndelivered.ts

```diff
@@ -99,31 +99,34 @@ const unsubscribeMessageUndeliveredHook = (z: ZObject, bundle: Bundle) => {
     Accept: "application/vnd.whispir.api-callback-v1+json",
   };
 
-  return z
-    .request({
-      url: `${bundle.authData.host}/callbacks/${hookId}`,
-      method: "DELETE",
-      headers: addHeaders(headers, bundle),
-    })
-    .then((response) => z.JSON.parse(response.data));
```

Comment:
removed `z.JSON.parse(response.data)` as the response does not have any data to be parsed